prime_factor/prime_factor.js

```diff
@@ -0,0 +1,21 @@
+function primeFactor (n) {
+	
+	for (let i = 2; i <= n; i++) {
+		if (n%i==0) {
+			console.log(i);
+			while(n%i==0) {
+        n=n/i;
+      }
+	  }
+	}
+}
+
+function main () {
+	let n=8;
+  console.log('Prime Factors are:');
+  for (let i = 2;i <= primeFactor(n);i++ ) {
```

Comment:
there should space after semicolon, as `for (let i = 2; i <= primeFactor(n); i++)`